### PR TITLE
Setup Koin in shared

### PR DESCRIPTION
**Pull request by MarekMacko**

### CODE REVIEW

File: shared/src/commonMain/kotlin/com/intive/picover/shared/coin/MainModule.kt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package com.intive.picover.shared.coin
```
Comment (gmatyszczak) [inline shared/src/commonMain/kotlin/com/intive/picover/shared/coin/MainModule.kt]: why `coin`, not `koin` 😄 ?

Comment (MarekMacko) [reply shared/src/commonMain/kotlin/com/intive/picover/shared/coin/MainModule.kt]: Good point 😄

Comment (MarekMacko) [reply shared/src/commonMain/kotlin/com/intive/picover/shared/coin/MainModule.kt]: Fixed

**Comment by gmatyszczak** [comment]
It's starting to look good! 😄

**Review by gmatyszczak — APPROVED** [review]

**Pull request merged**